### PR TITLE
Bug 2000768: Fix 4 Quick Starts

### PR DESCRIPTION
**Pull request by serenamarie125**
This fixes https://bugzilla.redhat.com/show_bug.cgi?id=2000768

**Fixed Issues** 

1. There are 4 Quick Starts which show the user how to View the associated code via the code decorator in the Topology view.  The current QS indicate that the code decorator always bring you to the associated git repository.
_Resolution_: The content now explains that the code decorator will either bring you to the associated git repo or to your IDE if Eclipse Che or CRW Operators are installed.

1. The deployment name was incorrect ( due to the fact that theses QS were initially copies of one another)
_Resolution_: Incorrect deployment names were updated.

1. The deployment name from the Node.js sample was renamed when moving from s2i samples to devfile samples, but the quick starts weren't updated
_Resolution_: changed workload references from **nodejs-sample** to **nodejs-basic** AND changed the name of the sample from **Node.js** to **Basic NodeJS**

1. In the Monitor your sample app QS, it referenced the ability to change from a particular workload to All workloads - this was removed in 4.9.
_Resolution_: removed reference to removed feature as well as included a step to allow users to discover available dashboards on the Observe -> Dashboard page

1. The add page was redesigned in 4.8, and the quick starts weren't updated appropriate ( From Git )
_Resolution_: Updated to reflect that Import from Git is now on the Git Repository card on the Add page

Animated gifs of updated quick starts:
- [Getting Started with a Sample - Animated gif](https://user-images.githubusercontent.com/12504403/131946036-a81ad4cd-989f-4b39-a5a9-a79c95667538.gif)
- [AddHealthChecks - Animated gif](https://user-images.githubusercontent.com/12504403/131946053-e62263c0-9d1e-4e6f-aa7f-54bf53c4b9e4.gif)
- [MonitorSampleApp - Animated gif](https://user-images.githubusercontent.com/12504403/131946067-fc48635d-9d09-493b-a6b9-7c89070bd767.gif)
- [GetStartedWithNode - Animated gif](https://user-images.githubusercontent.com/12504403/131946095-ba695508-e2ef-4fc2-a4d1-6e8677ab7729.gif)
- [GetStartedWithQuarkusS2i - Animated gif](https://user-images.githubusercontent.com/12504403/131946098-8a2b3f4c-2e4c-431e-a33c-14b8750e6fb8.gif)
- [GetStartedWithSpring - Animated gif](https://user-images.githubusercontent.com/12504403/131946099-d9c2811f-6ac2-4292-a9ca-26332951cf88.gif)

### CODE REVIEW

**Comment by serenamarie125** [comment]
NOTE - I need to add associated screenshots.

**Comment by serenamarie125** [comment]
@andrewballantyne @spadgett FYI

File: quickstarts/sample-application-quickstart.yaml [hunk 1]
```diff
@@ -55,23 +55,31 @@ spec:
...
       summary:
-        success: You have viewed the associated Git repo.
         failed: Try the steps again.
+        success: You viewed the code associated with the **nodejs-basic** deployment.
+
     - title: View the pod status
       description: |-
         ### To view the pod status:
-        1. Click on the browser tab with OpenShift.  Notice that the **nodejs-sample** deployment has a pod donut imposed on the circle, representing the pod status
+        1. Click on the browser tab with OpenShift.  Notice that the **nodejs-basic** deployment has a pod donut imposed on the circle, representing the pod status
```
Comment (jhadvig) [inline quickstarts/sample-application-quickstart.yaml:82]:
```suggestion
        1. Click on the browser tab with OpenShift.  Notice that the **nodejs-basic** deployment has a pod donut imposed on the circle, representing the pod status.
```

**Review by jhadvig — CHANGES_REQUESTED** [review]
@serenamarie125 thanks for the PR!
Found one small nit.

**Review by spadgett — APPROVED** [review]
/approve
/assign @andrewballantyne

File: quickstarts/sample-application-quickstart.yaml (hunk 1 above)
Comment (serenamarie125) [reply quickstarts/sample-application-quickstart.yaml]: Thanks @jhadvig! updated the reference to **nodejs-sample** and added the period.

**Comment by jhadvig** [comment]
@serenamarie125   forgot to mention but could you pls squash your PR into single commit. Thanks

**Review by andrewballantyne — APPROVED** [review]
/lgtm

**Comment by serenamarie125** [comment]
> @serenamarie125 forgot to mention but could you pls squash your PR into single commit. Thanks

Done @jhadvig

**Pull request merged**